ci/image/Dockerfile.nodejs-centos7-fips-test

```diff
@@ -35,29 +35,49 @@ SHELL [ "/usr/bin/scl", "enable", "devtoolset-8"]
 
 # node-fips environment variables
 ENV NODE_HOME $HOME/node
-ENV NODEJS_VERSION 14.0.0
-ENV FIPSDIR $HOME/install-openssl-fips
```

Comment:
Don't need FIPSDIR anymore?

Reply:
Not anymore, the new node version requires setting the openssl lib/include directory location instead of the openssl fips install directory when configuring